### PR TITLE
projects census

### LINKED ISSUE #44: List what is sandbox, incubating, official support
**What to document**

We need a place to use as a truth where we list what is the status of the various projects/subprojects.

For example:

- falcosecurity/falco: official support
- falcosecurity/kilt: incubating
- and so on

In some cases, some features or release artifacts of certain projects might not have the same level of the project itself. For example ECR images for Falco are not official support yet.

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**

/kind cleanup

/kind documentation

**What this PR does / why we need it**:

We needed a source of truth for the status of our projects, thus I have done the first census. To include all repositories in the same place, I have also introduced a section for those repositories with a special meaning (e.g., `falcosecurity/.github`). 

**Which issue(s) this PR fixes**:

Fixes #44

**Special notes for your reviewer**:

Since the importance of this list, I'd want to invite all the maintainers to double-check that everything is accurate, please.
cc @falcosecurity/evolution-maintainers @falcosecurity/falco-maintainers

### CODE REVIEW

**Comment by maxgio92** [comment]
This is great @leogr!

**Comment by maxgio92** [comment]
What about putting this in a special repository? (github.com/falcosecurity/falcosecurity)
So that you can see the index on the home of the falcosecurity Github profile.

**Review by cpanato — APPROVED** [review]
looks great! thanks for doing this

/lgtm

/hold for more eyes

File: README.md [hunk 1]
```diff
@@ -35,9 +35,50 @@ This status is assigned as needed, and can best be measured by the need to cut a
...
+
+
+| Project | Status | Note |
+| --- | --- | --- |
+| [falcosecurity/falco](https://github.com/falcosecurity/falco) | Official support | The list of official artifacts can be found within the [official documentation](https://falco.org/docs/download/). These artifacts will be refined and amended as per the [Falco Artifacts Scope - Part 2](https://github.com/falcosecurity/falco/blob/master/proposals/20200506-artifacts-scope-part-2.md).  |
+| [falcosecurity/falco-website](https://github.com/falcosecurity/falco-website) | Official support | | 
+| [falcosecurity/libs](https://github.com/falcosecurity/libs) | Official support | Artifacts will be avialable once [Versioning and release process of the libs artifacts](https://github.com/falcosecurity/libs/blob/master/proposals/20210524-versioning-and-release-of-the-libs-artifacts.md) proposal is fully implemented. |
```
Comment (leodido) [inline README.md:47]:
```suggestion
| [falcosecurity/libs](https://github.com/falcosecurity/libs) | Official support | Artifacts will be available once [Versioning and release process of the libs artifacts](https://github.com/falcosecurity/libs/blob/master/proposals/20210524-versioning-and-release-of-the-libs-artifacts.md) proposal is fully implemented. |
```

**Review by leodido — COMMENTED** [review]
Left a suggestion for a typo!

File: README.md (hunk 1 above)
Comment (leogr) [reply README.md]: Thank you! Fixed

**Review by maxgio92 — APPROVED** [review]
/approve

**Pull request merged**